### DISCUSSION TITLE
Failing to download forge.

### DISCUSSION THREAD

**Discussion by WheelSnipeChelly** [itzg/docker-minecraft-server#1951] [Q&A] [answered]
version: '3.7'

services:
  minecraft:
    image: itzg/minecraft-server
    container_name: minecraft
    environment:
      EULA: "TRUE"
      MEMORY: 32G
      TYPE: FORGE
      VERSION: 1.18.2
      FORGE_VERSION: recommended
      FORGE_FORCE_REINSTALL: "true"
    volumes:
      - minecraft_data:/data
      - /home/voldemort/mods:/data/mods
    ports:
      - "25565:25565"

volumes:
  minecraft_data:
  minecraft_mods:

Here are the error logs. https://pastebin.com/YKr2WEXY

Sorry I'm new to docker containers in general, but I had this running a few days back and my only issue was I didn't have enough memory allocated for a modded server. Went to remake the server and now I am running into this issue. Also note, that even if I remove 'FORGE_VERSION: recommended it gives the same error. Any help appreciated, thanks.

**Comment by itzg** [comment, marked as answer]
It look like a networking issue from this part of the logs

```
connection timed out: files.minecraftforge.net/51.79.83.165:443
```

It might even be temporary and will start working on its own.

**Reply by WheelSnipeChelly** [reply]
It was a really dumb mistake on my part. I didn't realize I had checked to install docker in the ubuntu server OS setup and then installed it manually. I'm assuming this was causing network issues. I sorted it all out and it's up and working now.

Thanks for the response.

**Reply by cdhagmann** [reply]
I think I did the same thing. How did you sort it out?
